.changes/nsis-languages.md

```diff
@@ -0,0 +1,8 @@
+---
+'tauri-bundler': 'minor'
```

Comment:
```suggestion
"tauri-bundler": patch
```

@FabianLars I based on your comment to do this. It should be more clear for the author.
Good review btw!